.github/workflows/release.yml

```diff
@@ -63,6 +63,9 @@ jobs:
       - name: Publish Plugin
         env:
           PUBLISH_TOKEN: ${{ secrets.PUBLISH_TOKEN }}
+          CERTIFICATE_CHAIN: ${{ secrets.CERTIFICATE_CHAIN }}
+          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+          PRIVATE_KEY_PASSWORD: ${{ secrets.PRIVATE_KEY_PASSWORD }}
```

Comment:
For this to work, we'll need to add GitHub actions secrets as described in https://plugins.jetbrains.com/docs/intellij/plugin-signing.html.